### ISSUE TITLE
compileDebugKotlin FAILED Unresolved reference: setShareUrl

### ISSUE THREAD

**Issue by KirillRodichevUtorg** [Netvent/storyly-mobile#293]
Android build fails

## Environment Details
**Storyly SDK Version:** 2.0.2
**React Native SDK Version:** 2.0.0
**Platform:** Android
## Expected Behavior

Build success

## Current Behavior

> Task :storyly-react-native:compileDebugKotlin FAILED
e: ../node_modules/storyly-react-native/android/src/main/java/com/appsamurai/storyly/reactnative/STStorylyManager.kt: (135, 103): Unresolved reference: setShareUrl

## Steps to Reproduce
1. yarn add storyly-react-native
2. set classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:1.6.20")
3. run build

if I comment out this line
```
storylyBundle.getString("storylyShareUrl")?.let { storylyConfigBuilder = storylyConfigBuilder.setShareUrl(it) }
```
the project builds

**Comment by KirillRodichevUtorg** [comment]
Same for iOS in line
```
if let shareUrl = json["storylyShareUrl"] as? String { storylyConfigBuilder = storylyConfigBuilder.setShareUrl(url: shareUrl) }
```

**Comment by yigcal** [comment]
Hello @KirillRodichevUtorg ,

We've released [storyly-react-native@2.0.1](https://github.com/Netvent/storyly-mobile/blob/master/react-native/storyly-react-native/RELEASENOTES.md#201-02082023).

Thanks
